### PR TITLE
Fix errors, warning on init load

### PR DESCRIPTION
**Pull request by CedrikNikita**
ref #703

### CODE REVIEW

File: src/components/MessageInput.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="message-input">
-    <Avatar :address="address" />
+    <Avatar :address="address || ''" />
```
Comment (davidyuk) [inline src/components/MessageInput.vue:3]: I think it is better to set `''` as a default value of `address` property of Avatar component instead.

Especially, if we assume that 
Screenshot 2020-08-17 at 21 03 46
is a valid avatar of an anonymous user.

File: src/components/layout/FooterSection.vue [hunk 2]
```diff
@@ -61,7 +61,7 @@
       class="login-footer"
     >
       <OutlinedButton
-        :href="addressDeepLink"
+        :href="addressDeepLink.toString()"
```
Comment (davidyuk) [inline src/components/layout/FooterSection.vue:64]: Technically, we can make it accept Url instance as well, but that component anyway is not enought flexible now, so I don't know how to handle it better. The best would be to make all buttons to accept full url, router location, click handler (displaying it as <a/button/RouterLink>) as we did in the [Base aepp](https://github.com/aeternity/aepp-base/blob/d17541574773e0b2db3fd1b66b1b0a4ea06ccf4a/src/components/ButtonPlain.vue).

**Comment by mradkov** [comment]
@CedrikNikita can you address the suggested changes by @davidyuk. Thanks

File: src/components/ButtonPlain.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  border: none;
+  background: transparent;
+  padding: 0;
+  text-decoration: none;
+  text-align: center;
+  font: inherit;
+  color: inherit;
+  cursor: pointer;
+
+  &:focus:not([data-focus-visible-added]) {
```
Comment (davidyuk) [inline src/components/ButtonPlain.vue:37]: This is for adding an outline when keyboard is used, it depends on https://www.npmjs.com/package/focus-visible package.

File: src/components/Link.vue [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  <RouterLink
+    v-if="useRouterLink"
+    class="ae-link"
+    :to="to"
+  >
+    <slot />
+  </RouterLink>
+  <a
+    v-else
+    class="ae-link"
```
Comment (davidyuk) [inline src/components/Link.vue:11]:
```suggestion
    class="link"
```
the same above

File: src/components/Link.vue [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  props: {
+    to: { type: [String, Object], required: true },
+  },
+  computed: {
+    isLinkOnSameHost() {
+      return typeof this.to === 'object'
+        || (new URL(this.to, window.location)).host === window.location.host;
+    },
+    useRouterLink() {
+      return this.$options.components.RouterLink && this.isLinkOnSameHost;
```
Comment (davidyuk) [inline src/components/Link.vue:31]: This was needed to be able to use it without VueRoute, it can be removed in this project

**Review by davidyuk — APPROVED** [review]

File: src/components/Link.vue [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <RouterLink
+    v-if="useRouterLink"
```
Comment (davidyuk) [inline src/components/Link.vue:3]:
```suggestion
    v-if="isLinkOnSameHost"
```

File: src/components/Link.vue [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    class="ae-link"
+    :to="to"
+  >
+    <slot />
+  </RouterLink>
+  <a
+    v-else
+    class="ae-link"
+    :href="to"
+    :target="isLinkOnSameHost ? '_self' : '_blank'"
```
Comment (davidyuk) [inline src/components/Link.vue:13]:
```suggestion
    target="_blank"
```

**Pull request merged**